apps/extension/hooks/use-farcaster-identity.tsx

```diff
@@ -73,7 +77,7 @@ async function fetchCreateSignedKeyRequest(requestBody: SignedKeyRequestBody) {
   return result
 }
 
-interface SignerResponse {
+type SignerResponse = Omit<SignedKeyRequestBody, "key"> & {
```

Comment:
can you explain the preference for `type`? (i am pretty much uninitiated in typescript and want to learn)

Reply:
the syntax is just easier when combining types